### PR TITLE
feat(html): Add HTML tests

### PR DESCRIPTION
**Pull request by leonardorpr**
I realized that we didn't have tests on this component, so I decided to add it. Makes sense?

### CODE REVIEW

**Comment by zenorocha** [comment]
Always good, thanks!

File: packages/html/package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
     "dev": "tsup src/index.ts --format esm,cjs --dts --external react --watch",
     "lint": "eslint",
     "clean": "rm -rf dist",
+    "test": "jest",
+    "test:watch": "jest --watch",
```
Comment (bukinoshita) [inline packages/html/package.json:19]: No need the watch command here

```suggestion
```

Comment (leonardorpr) [reply packages/html/package.json:19]: Ok, I'll remove

Comment (leonardorpr) [reply packages/html/package.json]: Done @bukinoshita

**Pull request merged**